SonarQube.TeamBuild.PreProcessor/Interfaces/ISonarQubeServer.cs

```diff
@@ -34,7 +34,7 @@ public interface ISonarQubeServer
         /// <summary>
         /// Get all the properties of a project
         /// </summary>
-        IDictionary<string, string> GetProperties(string projectKey, ILogger logger);
+        IDictionary<string, string> GetProperties(string projectKey);
```

Comment:
This was the only method on the interface that required a logger. To be consistent I've removed it.